README.md

```diff
@@ -3,27 +3,298 @@
+# Introduction
+The OpenSRP Path is an app that handles infant records from child birth to 5 years of age. This mainly includes monitoring child weight, vaccination schedules and other supplements that have been proven to prevent diseases and/or, improve infant growth and health.
 
-Then, you can run `mvn clean install` in the main directory.
+It's main focus is **Infant Health**
 
-Tips and tricks (to be completed):
-=================================
 
-* How to setup your Android SDK so that Maven finds it: http://pivotal.github.com/robolectric/maven-quick-start.html
+## Features
 
-* Adding an external Android library (apklib) as a submodule, and making it work with both Maven and IntelliJ. TODO: Write about `mvn clean` trick.
+The OpenSRP Path App enables providers to:
+ 1. Record patient immunisations
+ 2. Register new patients
+ 3. Record vaccine stock
+ 4. Record patient weights over time
+ 5. Review patient immunisation & weight history
+ 6. Determine patient health status based on weight
+ 7. Determine patient health vulnerability based on adherance to immunisation schedule
+ 8. Determine patient health from **above factors**
+ 9. Record side-effects caused by treatments
+ 10. Follow-up with patients in the field
+ 11. Perform stock management of vaccines
+ 12. Submit HIA 2 forms monthly
+ 13. Verify HIA 2 forms monthly
+ 14. Administer to patients from different zones
+
+# Android Studio Setup - (Developer Setup)
+
+## Pre-requisites
+
+1. Make sure you have Java 1.7 to 1.8 installed
+2. Make sure you have Android Studio installed or [download it from here](https://developer.android.com/studio/index.html)
+
+## Testing devices
+
+1. Use a physical Android device to run the app
+2. Use the Android Emulator that comes with the Android Studio installation (Slow & not advisable)
+3. Use Genymotion Android Emulator
+	* Go [here](https://www.genymotion.com/) and register for genymotion account if none. Free accounts have limitations which are not counter-productive
+	* Download your OS Version of VirtualBox at [here](https://www.virtualbox.org/wiki/Downloads)
+	* Install VirtualBox
+	* Download Genymotion & Install it
+	* Sign in to the genymotion app
+	* Create a new Genymotion Virtual Device 
+		* **Preferrable & Stable Choice** - API 22(Android 5.1.0), Screen size of around 800 X 1280, 1024 MB Memory --> eg. Google Nexus 7, Google Nexus 5
+
+## Cont.. Android Studio Setup
+
+1. Import the project into Android Studio by: **Import a gradle project** option
+   _All the plugins required are explicitly stated therefore can work with any Android Studio version - Just enable it to download any packages not available offline_
+1. Open Genymotion and Run the Virtual Device created previously.
+1. Run the app on Android Studio and chose the Genymotion Emulator
+
+
+
+
+## App Walkthrough (End-User)
+
+1. **(Optional)** Open the app, open the app menu > Settings > Change url to ```http://46.101.51.199:8080/opensrp```
```

Comment:
should we note that this webserver may not be up? i.e. we don't want to be responsible for keeping this up forever

Reply:
Noted